src/pages/create-or-edit-lesson/CreateOrEditLesson.styles.ts

```diff
@@ -8,46 +21,54 @@ export const styles = {
     gap: '24px',
     p: { sm: '40px 36px', md: '40px 72px' }
   },
+
   input: {
     style: {
       padding: 0
     }
   },
-  descriptionLabel: {
-    shrink: false,
-    sx: { typography: TypographyVariantEnum.Body1, top: -21 }
-  },
+
   divider: {
     color: 'primary.300'
   },
-  titleInput: (isDirty: boolean) => ({
-    disableUnderline: true,
+
+  titleLabel: (value: string) => ({
+    shrink: false,
     style: {
-      fontSize: '35px',
-      fontWeight: 500,
-      marginTop: 0,
-      maxHeight: isDirty ? 'unset' : '35px'
+      visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,
+      color: palette.primary[300],
+      top: -23,
+      ...inputFontSize
     }
   }),
-  descriptionInput: (isDirty: boolean) => ({
-    disableUnderline: true,
+
+  titleInput: {
+    style: inputFontSize,
+    disableUnderline: true
+  },
+
+  descriptionLabel: (value: string) => ({
+    shrink: false,
     style: {
-      fontSize: '16px',
-      padding: '0',
-      marginTop: 0,
-      maxHeight: isDirty ? 'unset' : '16px'
+      visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,
+      color: palette.primary[300],
```

Comment:
The following files are duplicated:
```jsx
shrink: false,
style: {
  visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,
  color: palette.primary[300],
```

Please consider reusing it

Reply:
Yes, I agree with you. 
I added it to the function.
fixed 

- [ ] fixed